packages/ui-proxy/src/constants.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2011-2018 Genestack Limited
+ * All Rights Reserved
+ * THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF GENESTACK LIMITED
+ * The copyright notice above does not evidence any
+ * actual or intended publication of such source code.
+ */
+
+module.exports = {
+    DEFAULT_LIVESYNC_PORT: 29272,
+    DEFAULT_PROXY_PORT: 3030,
+    DEFAULT_STATIC_PORT: 3459,
+    DEFAULT_REPLACE_SERVER: 'https://internal-dev.genestack.com'
```

Comment:
Default server should be `https://platform.genestack.org` – this is a public library, so its users have no access to internal-dev.